checks/docker/latest_tag_test.rego

```diff
@@ -189,3 +189,124 @@ test_multi_stage_no_tag_denied if {
 	count(r) == 1
 	r[_].msg == "Specify a tag in the 'FROM' statement for image 'alpine'"
 }
+
+test_deny_latest_tag_ref_to_global_arg_with_default_value if {
+	r := deny with input as {"Stages": [
+		{"Name": "", "Commands": [{
+			"Cmd": "arg",
+			"Value": ["TAG=\"latest\""],
+		}]},
+		{"Name": "foo:${TAG}", "Commands": [{
+			"Cmd": "from",
+			"Value": ["foo:${TAG}"],
+		}]},
+	]}
+
+	count(r) == 1
+	r[_].msg == "Specify a tag in the 'FROM' statement for image 'foo'"
+}
+
+test_allow_tag_ref_to_global_arg_without_default_value if {
+	r := deny with input as {"Stages": [
+		{"Name": "", "Commands": [{
+			"Cmd": "arg",
+			"Value": ["TAG"],
```

Comment:
If the tag doesn't have a value doesn't it default to latest? Or this test is testing another case?

Reply:
Ah I see the difference now. Thanks!